### PR TITLE
GSLA - move file with older creation date to ERROR_DIR

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

File: aodndata/gsla/handler.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from datetime import datetime
 
 from aodncore.pipeline import HandlerBase, FileType, PipelineFilePublishType, PipelineFile
-from aodncore.pipeline.exceptions import InvalidFileNameError, InvalidInputFileError, InvalidPathFunctionError
```
Comment (ggalibert) [inline aodndata/gsla/handler.py]: Is this change intended?

Comment (lbesnard) [reply aodndata/gsla/handler.py]: yes, because it wasn't used anyway
https://github.com/aodn/python-aodndata/blob/master/aodndata/gsla/handler.py
